sites/public/src/pages/listing/ext/[id].tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useContext } from "react"
```

Comment:
FYI this is copypasta from https://github.com/metrotranscom/Doorway/blob/325799dd0cce19f3eac51a56aa9ea8a51cbbb953/sites/public/src/pages/listing/%5Bid%5D/%5Bslug%5D.tsx#L103. But there isn't an easy way for git to preserve blame -_-